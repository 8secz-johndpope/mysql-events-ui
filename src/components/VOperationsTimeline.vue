<script>
export default {
  name: 'operations-timeline',
}
</script>

<template>
  <div>
    OperationsTimeline
  </div>
</template>
