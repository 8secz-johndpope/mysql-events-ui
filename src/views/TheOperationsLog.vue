<script>
import VOperationsTimeline from '@/components/VOperationsTimeline'
export default {
  name: 'operations-log',
  components: {
    VOperationsTimeline,
  }
}
</script>

<template>
  <div>
    <VOperationsTimeline />
  </div>
</template>