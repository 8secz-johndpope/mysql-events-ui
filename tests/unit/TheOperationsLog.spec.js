import { shallowMount } from '@vue/test-utils'
import TheOperationsLog from '@/views/TheOperationsLog'
import VOperationsTimeline from '@/components/VOperationsTimeline'

describe('TheOperationsLog', () => {
  const build = () => {
    const wrapper = shallowMount(TheOperationsLog)

    return {
      wrapper,
      operationsTimeline: () => wrapper.find(VOperationsTimeline)
    }
  }

  it('renders the component correctly', () => {
    // arranje
    const { wrapper } = build()

    // assert
    expect(wrapper.html()).toMatchSnapshot()
  })

  it('renders main components', () => {
    // arranje
    const { operationsTimeline } = build()

    // assert
    expect(operationsTimeline().exists()).toBe(true)
  })
})