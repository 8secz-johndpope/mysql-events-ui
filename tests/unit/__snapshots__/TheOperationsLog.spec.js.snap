// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TheOperationsLog renders the component correctly 1`] = `
<div>
  <operations-timeline-stub></operations-timeline-stub>
</div>
`;
